packages/labs/src/components/popover/popperUtils.ts

```diff
@@ -0,0 +1,103 @@
+import * as PopperJS from "popper.js";
+
+//
+// Popper Placement Utils
+//
+
+/** Converts a full placement to one of the four positions by stripping text after the `-`. */
+export function getPosition(placement: PopperJS.Placement) {
+    return placement.split("-")[0] as PopperJS.Position;
```

Comment:
absolutely not! I've always disliked the Position enum cuz it's really only usable as `Position.BOTTOM_LEFT` which is sooooo long. `Placement` is a union of string literals like `"left"` or `"top-start"` so they're short and effective, and the naming scheme is much more logical.

who among us _really_ knows the difference between `BOTTOM_LEFT` and `LEFT_BOTTOM`?
but `"bottom-start"` and `"left-end"`, those are terms I can work with!